src/views/Simulation/Ressources/Types.vue

```diff
@@ -13,7 +14,7 @@ export default {
   },
   computed: {
     individu: function() {
-      return Individu.find(this.$store.state.situation, this.$route.params.role, this.$route.params.id)
+      return Individu.find(this.$store.state.situation, _.get(this.$route.params.id.split('_'), '0', this.$route.params.id), this.$route.params.id)
```

Comment:
```suggestion
      return Individu.find(this.$store.state.situation, this.$route.params.id.split('_')[0], this.$route.params.id)
```
fait la même chose sans renforcé la dépendance à `lodash`.